package.json

```diff
@@ -30,37 +25,37 @@
       "name": "Brian Runnells"
     }
   ],
-  "license": "MIT",
   "dependencies": {
     "ember-cli-babel": "^6.10.0",
     "ember-cli-htmlbars": "^2.0.1",
     "ember-runtime-enumerable-includes-polyfill": "^2.0.0"
   },
   "devDependencies": {
     "@types/ember": "^2.8.4",
-    "broccoli-asset-rev": "^2.4.5",
-    "ember-cli": "~3.0.2",
+    "broccoli-asset-rev": "^2.7.0",
+    "ember-cli": "~3.3.0",
     "ember-cli-dependency-checker": "^2.0.0",
     "ember-cli-eslint": "^4.2.1",
     "ember-cli-github-pages": "0.0.6",
     "ember-cli-htmlbars-inline-precompile": "^1.0.0",
     "ember-cli-inject-live-reload": "^1.4.1",
-    "ember-cli-qunit": "^4.1.1",
+    "ember-cli-qunit": "^4.3.2",
     "ember-cli-shims": "^1.2.0",
     "ember-cli-sri": "^2.1.0",
     "ember-cli-uglify": "^2.0.0",
     "ember-cli-version-checker": "^2.1.0",
     "ember-disable-prototype-extensions": "^1.1.2",
     "ember-export-application-global": "^2.0.0",
-    "ember-load-initializers": "^1.0.0",
+    "ember-load-initializers": "^1.1.0",
     "ember-maybe-import-regenerator": "^0.1.6",
     "ember-resolver": "^4.0.0",
-    "ember-source": "~3.0.0",
+    "ember-source": "~3.3.0",
     "ember-source-channel-url": "^1.0.1",
     "ember-try": "^0.2.23",
     "eslint-plugin-ember": "^5.0.0",
-    "eslint-plugin-node": "^5.2.1",
-    "loader.js": "^4.2.3"
+    "eslint-plugin-node": "^6.0.1",
+    "loader.js": "^4.2.3",
+    "qunit-dom": "^0.6.2"
```

Comment:
I know this is part of the default package now, but unless we start moving the tests over to use this, we can probably remove this dependency.  Maybe add it in a later PR?

Reply:
good point, i'll remove this